packages/world/package.json

```diff
@@ -11,6 +11,7 @@
   "type": "module",
   "exports": {
     ".": "./dist/index.js",
+    "./actions": "./dist/actions.js",
```

Comment:
I don't think we need to make this a separate export now that we don't have global side effects on the global export. We can just export this action via `/ts/exports/index.ts` (or `ts/exports/internal.ts` for now until we want to make it "public")